pubsub/pstest/fake.go

```diff
@@ -617,6 +620,14 @@ func (s *GServer) UpdateSubscription(_ context.Context, req *pb.UpdateSubscripti
 				}
 			}
 
+		case "cloud_storage_config":
+			sub.proto.CloudStorageConfig = req.GetSubscription().GetCloudStorageConfig()
```

Comment:
there's some invariant here that ensures req.GetSubscription() doesn't return nil?

Reply:
Correct, it's checked in the first lines above